src/Toolkit/Toolkit.Maui/BookmarksView/BookmarksView.cs

```diff
@@ -192,16 +170,36 @@ private void SelectAndNavigateToBookmark(Bookmark bookmark)
     /// <summary>
     /// Handles selection on the underlying list view.
     /// </summary>
-    private void Internal_bookmarkSelected(object? sender, SelectedItemChangedEventArgs e)
+    private void Internal_bookmarkSelected(object? sender, SelectionChangedEventArgs e)
+    {
+        if (sender is CollectionView cv)
+        {
+            if (cv.SelectedItem is Bookmark item)
+            {
+                SelectAndNavigateToBookmark(item);
+            }
+            cv.ClearValue(CollectionView.SelectedItemProperty);
+        }
+    }
+
+    private void UpdateListView()
     {
-        if (e.SelectedItem is Bookmark bm)
+        var collection = (CollectionView)GetTemplateChild(_presentingViewName);
+        if (collection != null)
         {
-            SelectAndNavigateToBookmark(bm);
```

Comment:
What happened to this line? What triggers the navigation event now?

Reply:
Navigation event is triggered by [same code](https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/pull/598/files#diff-b515b77dae873f62a7192e60f5ac03fd034dd3bb71c44393ca4cef9adc673752R179). It's just Git changes are skewed a little.